src/plots/gl2d/scene2d.js

```diff
@@ -667,3 +675,18 @@ proto.hoverFormatter = function(axisName, val) {
     var axis = this[axisName];
     return Axes.tickText(axis, axis.c2l(val), 'hover').text;
 };
+
+proto.castHoverOption = function(trace, ptNumber, attr) {
```

Comment:
It is possible. I didn't do it because as brought up in https://github.com/plotly/plotly.js/pull/1582#issuecomment-297115980, the _real_ solution would be to replace `Fx.loneHover` by something more similar to `Fx.hover` in gl2d and gl3d. But, for the time being, might as well put this logic something in the `Fx` module object.